frontend/src/Frontend/Wallet.hs

```diff
@@ -140,6 +141,21 @@ findFirstVanityAccount predicate as = do
   (c, va) <- Map.lookupMin $ Map.filter predicate cm
   pure (n, c, va)
 
+findAccountByName
+  :: Reflex t
+  => Wallet key t
+  -> NetworkName
+  -> ChainId
+  -> AccountName
+  -> Dynamic t (Maybe (Some AccountRef))
+findAccountByName w netname cid n = ffor (_wallet_accounts w) $ \accStorage ->
+  Map.lookup netname (unAccountStorage accStorage) >>=
+    fmap getFirst . getOption . foldAccounts g
+  where
+    g acc@(aref :=> _) = if accountToName acc == n && accountChain acc == cid
+      then Option . Just . First $ Some aref
+      else Option Nothing
```

Comment:
[Option](https://hackage.haskell.org/package/base-4.12.0.0/docs/Data-Semigroup.html#t:Option)'s getting deprecated in ghc 8.8 and removed in 8.10. I think `Alt Maybe` would work here